federatorai/README.md

```diff
@@ -0,0 +1,52 @@
+# Agent Check: federatorai
```

Comment:
Is this an agent check-based integration? IIRC Federator.ai uploads the metrics directly to Datadog, and hence the Datadog-agent itself is not involved in the integration (i.e. it can work without the Datadog Agent per-se)

Reply:
already change to " # Federator.ai Integration"